Packs/CadoResponse/Integrations/CadoResponse/CadoResponse_description.md

```diff
@@ -0,0 +1,8 @@
+## BaseIntegration Help
```

Comment:
if you could please provide information about the points mentioned in the file (like obtaining API key and etc)
then please do (not an obligation)